tasks/compare-compiled-output.js

```diff
@@ -480,11 +493,11 @@ async function main(
 
 	if (!hasAnyChange) {
 		log.write(`\n${"✓".green}  No changes detected.\n`);
-		return Promise.resolve();
```

Comment:
Removing this lets the build continue with the web view whether changes are detected or not.